cockroachdb/src/jepsen/cockroach/runner.clj

```diff
@@ -28,6 +29,7 @@
    "bank-multitable"      bank/multitable-test
    "comments"             comments/test
    "register"             register/test
+   "multiregister"        multiregister/test
```

Comment:
you called it "multi-register" in the README

Reply:
Done.